core/dbt/cli/main.py

```diff
@@ -52,17 +55,30 @@ def cli(ctx, **kwargs):
     """An ELT tool for managing your SQL transformations and data models.
     For more documentation on these commands, visit: docs.getdbt.com
     """
-    incomplete_flags = Flags()
+    flags = Flags(invoked_subcommand=globals()[ctx.invoked_subcommand])
+
+    dbt.tracking.initialize_from_flags(flags)
```

Comment:
We're only using two flag values-- for better visibility can we pull those values out specifically?

`initalize_from_flags(
    flags.SEND_ANONYMOUS_USAGE_STATS,
    flags.PROFILES_DIR
)`